### ISSUE TITLE
Create a website

### ISSUE THREAD

**Issue by THPubs** [99xt/wp-artisan#3]
What if we create a simple static website for the tool? 😃 Might also need a logo!

**Comment by thinkholic** [comment]
@THPubs yeah, we will need something like that eventually once all the APIs are done, let's keep it for later :)

**Comment by THPubs** [comment]
@thinkholic Great! Yea better wait until all the base is set up.
